app/models/league.rb

```diff
@@ -2,7 +2,7 @@ class League < ActiveRecord::Base
   INSTANCE_ID = 1
 
   has_many :rules
-  enum theme: [:default, :classic]
+  enum theme: [:default, :classic, :oxford]
```

Comment:
This works but it'd be nice if there was some Rails-style convention over configuration.

Forex, themes could be packaged as zips with a set structure and dropped into a `themes` directory. The application'd then scan and find these themes, installing them and making them discoverable automatically.

Nice to have, sure. :hibiscus: